Description: Avoid invoking regression tests as part of normal build
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: no
Last-Update: 2011-08-20

--- a/Cakefile
+++ b/Cakefile
@@ -169,8 +169,6 @@
   unless process.env.MINIFY is 'false'
     {code} = require('uglify-js').minify code, fromString: true
   fs.writeFileSync 'extras/coffee-script.js', header + '\n' + code
-  console.log "built ... running browser tests:"
-  invoke 'test:browser'
 
 
 task 'doc:site', 'watch and continually rebuild the documentation for the website', ->
